### PR TITLE
feature file for story ER-50

### PR DESCRIPTION
**Pull request by AshwiniPKadam**
Hii team,
please review feature file and let me know any changes required.

### CODE REVIEW

File: src/test/resources/Features/Orderprod.feature [hunk 1]
```diff
@@ -0,0 +1,55 @@
+
+@Orderprod
+Feature: Order products
+  I want to use this template for my feature file
+
+  @tag1
+  Scenario: Title of your scenario
+    Given I want to write a step with precondition  +++
```
Comment (abhilashawagh) [inline src/test/resources/Features/Orderprod.feature]: Remove symbols +++ at Given statement

File: src/test/resources/Features/Orderprod.feature [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  Scenario: Title of your scenario
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+    
+     @tag4wr
```
Comment (abhilashawagh) [inline src/test/resources/Features/Orderprod.feature]: Remove alphabets wr at tag4.

**Review by abhilashawagh — COMMENTED** [review]
please make associated changes in feature file for ER-50 story

File: src/test/resources/Features/Orderprod.feature (hunk 1 above)
Comment (AshwiniPKadam) [reply src/test/resources/Features/Orderprod.feature]: changes done.

**Review by AshwiniPKadam — COMMENTED** [review]
corresponding changes done please check and confirm.

**Comment by abhilashawagh** [comment]
Alright, now you can merge it to master.
Thanku.

**Pull request merged**